pyproject.toml

```diff
@@ -179,6 +180,10 @@ select = [
 ]
 ignore = [
     "ISC001",   # flake8-implicit-str-concat: single-line-implicit-string-concatenation  # incompatible with the formatter
+    "S105",     # flake8-bandit: hardcoded password
+    "S106",     # flake8-bandit: hardcoded password
+    "S603",     # flake8-bandit: subprocess-without-shell-equals-true
```

Comment:
This rule is strange. It conflicts with S602:

- [S602](https://docs.astral.sh/ruff/rules/S602): `subprocess.run("foo", shell=True)` is discouraged
- [S603](https://docs.astral.sh/ruff/rules/S603): `subprocess.run("foo", shell=False)` is discouraged

After some research, I found that S602 is to be preferred, and S603 should best be ignored

Reply:
yea, S603 doesn't seem too great: https://github.com/PyCQA/bandit/issues/333